dashboard/app/coverage.go

```diff
@@ -216,8 +216,10 @@ func handleHeatmap(c context.Context, w http.ResponseWriter, hdr *uiHeader, p *c
 	}
 	ssService := getNsConfig(c, hdr.Namespace).Subsystems.Service
 	var subsystems []string
-	for _, ss := range ssService.List() {
-		subsystems = append(subsystems, ss.Name)
+	if ssService != nil {
```

Comment:
Nit: you could do `if ssService := getNsConfig(c, hdr.Namespace).Subsystems.Service; sService != nil {` to reduce the scope of the variable.

Reply:
@a-nogikh ptal

Reply:
> I'm not aware of any robot that automatically simplifies code.

I have the tooling on my PC that do the actual simplification. Feel free to visit us any time to feel the same excitement :)